### DISCUSSION TITLE
Stop Chromium stealing focus upon new tabs opening

### DISCUSSION THREAD

**Discussion by My-Name-Is-Adam** [jef/streetmerchant#2245] [Help]
Not sure if there is a way for this to happen? 

I've tried so many different ways. 

I would run headless but am unable to do that as I get captchas for Amazon. 

I can't really use a Virtual Machine because I want the product to open in my default browser when stock is found to purchase. 

Thank you. 

PS. I am using Windows 10.

**Comment by ArmaanCPro** [comment]
So sadly I do not have a solution. In fact, I am also suffering from the same exact problems as you. I also want it to open in my default browser (Brave). If you found a solution can you tell me? But, I think you can just drag the tab out of the main browser window and plop it back in while using headful. It is a scuffed solution but it's better than nothing. (Or maybe it isn't)
